modules/post/multi/manage/shell_to_meterpreter.rb

```diff
@@ -45,11 +45,17 @@ def initialize(info = {})
       OptString.new('BOURNE_PATH',
                     [false, 'Remote path to drop binary']),
       OptString.new('BOURNE_FILE',
-                    [false, 'Remote filename to use for dropped binary'])
+                    [false, 'Remote filename to use for dropped binary']),
+      OptInt.new('COMMAND_TIMEOUT',
+                 [true, 'How long to wait (in seconds) for a result when executing a command on the remote machine.', 15]),
     ])
     deregister_options('PERSIST', 'PSH_OLD_METHOD', 'RUN_WOW64')
   end
 
+  def command_timeout
+    datastore['COMMAND_TIMEOUT'] || 15
```

Comment:
The default is already in the `OptInt`- do we need this default here as well? 👀 

Or is this fixing an edgease where someone explicitly calls `unset COMMAND_TIMEOUT` ?